README.md

```diff
@@ -1,9 +1,9 @@
 # APIS EventManager
 [![Build Status](https://travis-ci.com/furthemore/APIS.svg?branch=production)](https://travis-ci.com/furthemore/APIS) [![Coverage Status](https://coveralls.io/repos/github/furthemore/APIS/badge.svg?branch=production)](https://coveralls.io/github/furthemore/APIS?branch=production)
 
-Data Model: http://www.gliffy.com/go/publish/10890219
+Data Model: http://www.gliffy.com/go/publish/10890219 (broken link?)
```

Comment:
I've generated a graph of the latest model and uploaded it [here](https://i.imgur.com/A4fPDf5.png).  We can update this with that link, but it may be worth-while to modify our CI to generate a new image for every commit automatically.

Reply:
I've made the corrections and re-pushed. I don't see a way to re-request the pull and don't know whether you'll get notification. So I'm hoping this comment gives you the ping you need! I'll also check in on Telegram (where I'm @JackNewhorse).